Sources/Apollo/UploadRequest.swift

```diff
@@ -69,7 +69,10 @@ open class UploadRequest<Operation: GraphQLOperation>: HTTPRequest<Operation> {
     // Make sure all fields for files are set to null, or the server won't look
     // for the files in the rest of the form data
     let fieldsForFiles = Set(files.map { $0.fieldName }).sorted()
-    var fields = self.requestBodyCreator.requestBody(for: operation, sendOperationIdentifiers: shouldSendOperationID)
+    var fields = self.requestBodyCreator.requestBody(for: operation,
```

Comment:
better to not change this.

Reply:
This has to change due to the removal of the default values for these parameters

Reply:
Hi @designatednerd , although the default values for these parameters has been removed, but I think it's a good idea to add a new extension method like this:
```
extension RequestBodyCreator {
    public func requestBody<Operation: GraphQLOperation>(for operation: Operation) {
        requestBody(operation, false, true, false)
    }
}
```

Reply:
I'll consider that, but I am concerned that would run a significant risk of reintroducing this same problem.